src/midiEditorCommands.cpp

```diff
@@ -1272,6 +1288,19 @@ void cmdMidiDeleteEvents(Command* command) {
 		translate_plural("{} event removed", "{} events removed", removed), removed));
 }
 
+void postMidiCopyEvents(int command) {
+	HWND editor = MIDIEditor_GetActive();
+	MediaItem_Take* take = MIDIEditor_GetTake(editor);
+	int count = countSelectedEvents (take);
+	if (fakeFocus != FOCUS_NOTE && fakeFocus != FOCUS_CC) {
+		fakeFocus = FOCUS_NOTE;
```

Comment:
Why do we want to reset fakeFocus to note here? That's inconsistent with a normal paste and I don't really follow the logic from a user perspective.